x-pack/plugins/fleet/server/services/epm/registry/index.ts

```diff
@@ -221,13 +221,14 @@ export async function fetchCategories(
   return fetchUrl(url.toString()).then(JSON.parse);
 }
 
-export async function getInfo(name: string, version: string, options: { cache?: boolean } = {}) {
-  const cache = options.cache ?? true;
+export async function getInfo(name: string, version: string) {
   return withPackageSpan('Fetch package info', async () => {
     let packageInfo = getPackageInfo({ name, version });
     if (!packageInfo) {
       packageInfo = await fetchInfo(name, version);
-      if (cache) setPackageInfo({ name, version, packageInfo });
+      // only cache registry pkg info for integration pkgs because
+      // input type packages must get their pkg info from the archive
+      if (packageInfo.type === 'integration') setPackageInfo({ name, version, packageInfo });
```

Comment:
While investigating a bug where the cache parameter wasn't passed, I decided it was more explicit to have this function know that we never cache registry info for input packages (because we have to get the info from the archive)